### PR TITLE
fix(welcome): fix value error on unmatched brancket

### PR DESCRIPTION
**Pull request by adekmaulana**
-> Change parsing error message to make it universal 

Change-Id: I0e5f341401c876a39150630da9bfbafd6aea8bd9

### CODE REVIEW

File: anjani/plugins/welcome.py [hunk 1]
```diff
@@ -265,7 +265,7 @@ async def cmd_setwelcome(self, ctx: command.Context) -> str:
 
         try:  # Try to build a text first to check message validity
             await self._build_text(welc_text or "", ctx.author, chat, self.bot.client)
-        except KeyError as e:
+        except (KeyError, ValueError) as e:
```
Comment (MrMissx) [inline anjani/plugins/welcome.py:268]: buat error message baru aja?
returnnya gini `raised from ValueError: Single '}' encountered in format string`

**Review by MrMissx — CHANGES_REQUESTED** [review]

File: anjani/plugins/welcome.py (hunk 1 above)
Comment (adekmaulana) [reply anjani/plugins/welcome.py:268]: hardcoded gitu? ntr kalo errornya lagi `'{'` kan masa diprintnya `'}'`

Comment (MrMissx) [reply anjani/plugins/welcome.py:268]: hmmm 'Failed when parsing text due to unmatched bracked' ?

Comment (adekmaulana) [reply anjani/plugins/welcome.py:268]: terlalu mengharcoded, kan bisa aja valuerror dari yg lain

**Review by MrMissx — APPROVED** [review]

**Pull request merged**